helix-term/src/commands/typed.rs

```diff
@@ -1091,20 +1091,30 @@ fn change_current_directory(
     }
 
     let dir = match args.first() {
-        Some(input_path) => {
+        Some(Cow::Borrowed("-")) => cx.editor.last_cwd.clone(),
```

Comment:
We can early return here to avoid the extra option handling:

```rust
Some(Cow::Borrowed("-")) => cx.editor.last_cwd.clone().ok_or(anyhow!("No previous working directory"))?,
```

Reply:
ah, thanks! this is much cleaner now